src/main/resources/view/PersonListCard.fxml

```diff
@@ -31,6 +31,7 @@
       <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
       <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
       <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
+      <Label fx:id="socialmedia" styleClass="cell_small_label" text="\$handle" />
```

Comment:
maybe we can consider if they have no social media handle to not show the tag? this seems like it will display an empty string?

Reply:
yep i did consider, but i wasnt sure how to keep adding and removing the ui label so i just settled with the empty string since ill be creating a "unnamed" handle for every person anyways